tests/test_environments/test_vlmevalkit_scoring.py

```
@@ -0,0 +1,71 @@
+
+
+def _score(response: str, expected: str = "B") -> tuple[float, bool, str | None]:
+    # ``_score_mcq`` never touches ``self``; calling it unbound keeps the test off
+    # ``__init__``, which requires the optional ``vlmeval`` package and a dataset download.
+    vr = VLMEvalKitEnvironment._score_mcq(None, response, expected, CHOICES)
+    details = vr.scoring_details
+    return vr.reward, details.get("unscored", False), details.get("unscored_reason")
+
+
+@pytest.mark.parametrize(
+    "response,reward,unscored,reason",
```

Ruff (0.16.3)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)